Tig 624 - Sidebar Nav

FayeMao opened the pull request:
# Description
[TIG-624](https://app.clickup.com/t/9009201449/TIG-624)
[Figma Link](https://www.figma.com/file/yZIMYFjiOsEpplrb6qr9nt/Create-to-Learn?type=design&node-id=38%3A8&mode=dev)

In this PR, I made the sidebar navigation as per the specifications in the Figma file. I also updated the footer accordingly to the Figma file. These are seen for desktop views only. The 'Create to Learn' logo now directs users to the /dashboard.

The side nav bar is seen from all pages, except on '/' and '/sign-up', which are both sign up pages.

## Before
Not logged in:
![image](https://github.com/TakingITGlobal/create-to-learn/assets/132935720/3f258a67-ccd1-42e2-ab4c-f56312efabea)


Logged in:
![image](https://github.com/TakingITGlobal/create-to-learn/assets/132935720/8270bdaa-5bcc-4093-8732-3f2a12eb7694)

Footer:
![image](https://github.com/TakingITGlobal/create-to-learn/assets/132935720/57998b9b-c984-460a-afba-b4bc57be6db1)


## After
When user is not logged in:
![image](https://github.com/TakingITGlobal/create-to-learn/assets/132935720/deb8a6c4-1626-4bc9-a1f7-77d0f27312c4)



When user is logged in:
![image](https://github.com/TakingITGlobal/create-to-learn/assets/132935720/e72cd09c-fd6c-4274-a4a3-35eb37629aba)

Footer:
![image](https://github.com/TakingITGlobal/create-to-learn/assets/132935720/f7596ce6-d9e0-45b7-83f5-c336a31d7226)

kirstenjvd commented on src/components/Footer.js:
```diff
@@ -96,78 +102,66 @@ function Footer(props) {
+          <div style = {{ width: '30%'}}>
+            <img
+              src={TakingITGlobalLogo}
+              alt="cultural-teachings-icon"
```
Alternative text you want to always be something that explains what the image is. In this case you would put "TakingITGlobal" because when a user selects this it would read it out that way.

kirstenjvd commented on src/components/Footer.js:
```diff
@@ -96,78 +102,66 @@ function Footer(props) {
+              width='100%'
+              height='100%'
+            />
           </div>
```
Same alternative text change but this would be changed to "Imagine Native"

kirstenjvd commented on src/components/Footer.js:
```diff
@@ -96,78 +102,66 @@ function Footer(props) {
+          <div style = {{ width: '20%'}}>
+            <img
+              src={CanadaLogo}
+              alt="cultural-teachings-icon"
```
Canada Logo

kirstenjvd commented on src/components/Footer.js:
```diff
@@ -96,78 +102,66 @@ function Footer(props) {
+          <div style = {{ width: '30%'}}>
+            <img
+              src={LenovoLogo}
+              alt="cultural-teachings-icon"
```
Lenova

kirstenjvd commented on src/components/SignUp.js:
```diff
@@ -20,10 +20,10 @@ function SignUp({ setDismissed = null, showDismissButton = false }) {
+        <Typography variant="h3">{t('sign-up-or')}</Typography>
         <img
           src={signupIMG}
-          alt="cultural-teachings-icon"
```
Since this image doesn't bring any extra information to the page this is a good instance for using role="presentation" meaning the image is just decorative and does give any helpful context to the page beyond aesthetics. In this case you just leave alt="" and add role="presentation"

kirstenjvd reviewed (commented):
This looks great @FayeMao ! 

A few fine tuned notes captured via screenshots and bullets. 

**Not Logged in:**

- Sign up button down by social icons
- Padding left of the links and "Create to learn" to all line up (logged in and not logged in)
- Create to Learn text left aligned

<img width="1097" alt="Screenshot 2024-02-19 at 6 13 29 PM" src="https://github.com/TakingITGlobal/create-to-learn/assets/6613598/33dfa9ac-9a2f-4e11-b121-afe3315a9bfa">


**Logged in:**

- Padding left of the links and "Create to learn" to all line up (logged in and not logged in)
- Remove the flex gap between nav items and only use the nav items padding so they aren't so spaced out
- Aim for the gaps between the text and "hr" to be equal above and below. 

<img width="792" alt="Screenshot 2024-02-19 at 6 27 00 PM" src="https://github.com/TakingITGlobal/create-to-learn/assets/6613598/b9fd721a-15c9-4913-87df-fd698680dea4">


**Footer**
- Alt text phrasing : see comments

**Sub Page updates because of change (screen shot 3)**
- Remove top "toolbar" and add top padding to all pages
- Review all page H1 headings and ensure they match between views on desktop (font size and space from top of screen)

<img width="1657" alt="Screenshot 2024-02-19 at 6 51 28 PM" src="https://github.com/TakingITGlobal/create-to-learn/assets/6613598/2d10e3cf-298b-4cb6-bbf7-f578761c4d1b">

FayeMao commented:
Finished the changes you listed above!

- Added sign in + sign up button
- Added a sign out button option
- Fixed CreatorSection title
- Added alt text to sign-up + footer 
- Added in padding (10px) to all sections to account for the removed header (spacing for the headings looked tight without the extra space, so I added a little bit of it)

# Current Views
![image](https://github.com/TakingITGlobal/create-to-learn/assets/132935720/1e8a8157-2690-4d60-a886-6d495524f1b5)
![image](https://github.com/TakingITGlobal/create-to-learn/assets/132935720/9e4d7558-7dae-44e6-8692-cee8754d96d4)
